src/index.test.ts

```diff
@@ -146,18 +156,23 @@ test('Response data can be deduplicated with graphql-deduplicator', async t => {
         book,
         ...Array(4).fill({
           __typename: book.__typename,
-          id: book.id
-        })
-      ]
-    }
+          id: book.id,
+        }),
+      ],
+    },
   })
 
+  t.deepEqual(deduplicated, deduplicatedWithHeader)
+
   t.deepEqual(body.data, inflate(deduplicated.data))
```

Comment:
If neither of deduplications worked, will the test pass?

Reply:
No, that one is just to check that both are the same, to check that data is properly deduplicated there's another check above

Reply:
Ah, sure! I somehow missed `t.deepEqual` above 😅